python/ray/data/dataset.py

```diff
@@ -1832,7 +1846,7 @@ def sum(
               containing the column-wise sum of the provided columns.
 
             If the dataset is empty, all values are null, or any value is null
-            AND ``ignore_nulls`` is ``False``, then the output will be None.
+            AND ``ignore_nulls`` is ``False``, then the output is ``None``.
```

Comment:
If the dataset is empty, all values are null. If ``ignore_null`` is ``False`` and any value is null, then the output is ``None``.